optmod/coptmod/node.h

```diff
@@ -14,6 +14,7 @@
 #include <stdlib.h>
 #include <math.h>
 #include "uthash.h"
+#include <inttypes.h>
```

Comment:
stdlib file includes go before local includes for consistency. I'll take car of this.

Reply:
Thanks Adam, I will do the same way in the future development.